scripts/sync_scheduled_agents.py

```
@@ -0,0 +1,151 @@
+
+        print(f"  {config['name']}")
+        print(f"    Schedule: {config['cron_expression']}")
+        print(f"    Model: {config.get('model', 'claude-sonnet-4-6')}")
+        print(f"    Enabled: {config.get('enabled', True)}")
+        if config.get("mcp_connector"):
+            print(f"    MCP: {config['mcp_connector']['name']} @ {config['mcp_connector']['url'][:50]}...")
+        print(f"    Prompt: {config['prompt'][:80]}...")
+        print()
+
+        if args.apply:
+            print(f"    To sync, use Claude Code:")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)